lnst/Recipes/ENRT/IpIpTunnelRecipe.py

```diff
@@ -59,14 +64,17 @@ class IpIpTunnelRecipe(MTUHWConfigMixin, PauseFramesHWConfigMixin, BaseTunnelRec
         type=StrParam, choices=set(["any", "ipip", "mplsip"]), mandatory=True
     )
 
+    net_ipv4 = IPv4NetworkParam(default="172.16.0.0/16")
```

Comment:
Wrong default here, should be `172.16.200.0/16` if we want to keep it the same

Reply:
Nice catch. Will update.

Reply:
Fixed.

Reply:
Actually once I fixed this, the things broke completely. The problem was:
```
  File "/root/virtualenvs/rhextensions-lnst-Xo1BSm3a-py3.9/lib/python3.9/site-packages/lnst/Recipes/ENRT/IpIpTunnelRecipe.py", line 22, in <module>
    class IpIpTunnelRecipe(MTUHWConfigMixin, PauseFramesHWConfigMixin, BaseTunnelRecipe):
  File "/root/virtualenvs/rhextensions-lnst-Xo1BSm3a-py3.9/lib/python3.9/site-packages/lnst/Recipes/ENRT/IpIpTunnelRecipe.py", line 67, in IpIpTunnelRecipe
    net_ipv4 = IPv4NetworkParam(default="172.16.200.0/16")
  File "/root/virtualenvs/rhextensions-lnst-Xo1BSm3a-py3.9/lib/python3.9/site-packages/lnst/Common/Parameters.py", line 224, in __init__
    super().__init__(IPv4Network, **kwargs)
  File "/root/virtualenvs/rhextensions-lnst-Xo1BSm3a-py3.9/lib/python3.9/site-packages/lnst/Common/Parameters.py", line 213, in __init__
    super(NetworkParam, self).__init__(**kwargs)
  File "/root/virtualenvs/rhextensions-lnst-Xo1BSm3a-py3.9/lib/python3.9/site-packages/lnst/Common/Parameters.py", line 41, in __init__
    self.default = self.type_check(kwargs["default"])
  File "/root/virtualenvs/rhextensions-lnst-Xo1BSm3a-py3.9/lib/python3.9/site-packages/lnst/Common/Parameters.py", line 217, in type_check
    return self._type(value)
  File "/usr/lib64/python3.9/ipaddress.py", line 1506, in __init__
    raise ValueError('%s has host bits set' % self)
ValueError: 172.16.200.0/16 has host bits set
```

Anyway, I've reverted the change and instead used `default_start` parameter of the `interface_address` generator.